Runtime/Env/EnvLoader.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json.Linq;
+using UnityEngine;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    public static class EnvLoader
+    {
+        private const string LOCAL_ENV_FILENAME = "env.local";
+        
+        public static string LoadVar(string varName, string defaultValue)
+        {
+            var jsonContent = Resources.Load<TextAsset>(LOCAL_ENV_FILENAME);
```

Comment:
The SDK wouldn't break, because this file isn't actually required and wouldn't be included for any developers (it's purely a local thing that would be GITIGNORED). Still though, your other comment makes more sense to me, even if it does mean we have to have the values hardcoded for partners too. The original goal was that partners would never see any dev env vars.

Reply:
RIght I think I misunderstood. So this EnvLoader.cs would only be in our Unity-SDK project not in Core? 🤔